src/fixate/drivers/ftdi.py

```diff
@@ -450,3 +441,21 @@ def _serial_shift_bit_bang(self, data, bytes_required, bb_mask):
         data_out.append(bb_mask + self.bb_latch ^ self.bb_inv_mask)
         data_out.append(bb_mask + self.bb_inv_mask)
         return data_out
+
+    def get_identity(self) -> str:
+        """Return identity string representing connected ftdi object"""
+        return f"{self.ftdi_description.decode()}"
+
```

Comment:
`return self.ftdi_description.decode()`?

Reply:
The ftdi_description comes from the raw FT Device info. Otherwise ends with the raw string as b'InstrumentX'

Reply:
Yes, previously had some some other details in there but can fix that now

Reply:
Done